packages/blocks/src/api/parser.js

```diff
@@ -242,7 +272,7 @@ export function parseWithAttributeSchema( innerHTML, attributeSchema ) {
  * @return {*} Attribute value.
  */
 export function getBlockAttribute( attributeKey, attributeSchema, innerHTML, commentAttributes ) {
-	const { type } = attributeSchema;
+	const { type, enum: enumSet } = attributeSchema;
```

Comment:
In case you're wondering:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Future_reserved_keywords